doc/dev/build_archived_sensfuncs.rst

```diff
@@ -31,4 +65,66 @@ How to build archived sensitivity functions
         cd sensfunc_files
         cp keck_deimos_*.fits PypeIt/pypeit/data/sensfuncs/
 
+How the DEIMOS sensitivity functions were built
+***********************************************
+
+The archived sensitivity functions for DEIMOS were created by from raw data
+gathered from scripted standard star observations. These are slitless
+observations that PypeIt cannot reduce on its own, so they are reduced using
+IDL scripts from Greg Wirth. 
+
+Examples of this data can be found in the `PypeIt dev-suite Google Drive`_ Google Drive.
+
++----------------------------------------------------------------+-------------------------------------------------------------------------------+
+| ``DEIMOS_Dev/Throughput/YYYY``                                 | Contains raw data files from that year.                                       |
++----------------------------------------------------------------+-------------------------------------------------------------------------------+
+| ``DEIMOS_Dev/Throughput/throughput_gdw/data_products/extract`` | Contains the reduced data from IDL used to generate the sensitivity functions.|
++----------------------------------------------------------------+-------------------------------------------------------------------------------+
+
+
+Converting the IDL reduced files to spec1d
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the PypeIt-development-suite repo,  the ``batch_convert_throughput_to_spec1d.py``
+script can create PypeIt spec1d files from the IDL output.
+
+By default, this script splits the IDL output into two detectors (DET03 and DET07). Because the IDL output isn't always
+even in length, it is neccessary to pass "pad" to the script. Alternatively the script can generate spec1d files by treating the IDL output as a 
+mosiac ('MSC03') by passing "nosplit":
+
+For example, if the files from Google Drive was downloaded to a
+``data_products`` directory. The below would create a ``spec1d_files`` directory with
+spec1d files created from the IDL output:
+
+   .. code-block:: bash
+
+      # Create two detector spec1d files
+      python $PYPEIT_DEV/dev_algorithms/fluxing/batch_convert_throughput_to_spec1d.py data_products/extract spec1d_files pad
+
+      # Create a mosaic spec1d file
+      python $PYPEIT_DEV/dev_algorithms/fluxing/batch_convert_throughput_to_spec1d.py data_products/extract spec1d_files nosplit
+
+The current DEIMOS archival sensfuncs were created using "pad" to split the output into two detectors.
+
+Creating the individual sensitivity functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The individual DEIMOS sensitivity functions were created with the following ``pypeit_sensfunc`` input file:
```

Comment:
Can you add here the reference to the documentation where the sensfunc input file is described? Should be :ref:`sensitivity_file`. 
It can be: "The individual DEIMOS sensitivity functions were created with the following :ref:`sensitivity_file`:"

Reply:
done